setup/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "setup-notation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Duffney/setup-notation.git"
```

Comment:
Shall we change the url to this Github repo?

Reply:
Done. :)